docs/reference/changelog.rst

```diff
@@ -16,6 +16,19 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release-2.12.0:
+
+2.12.0 (2025-06-06)
+-------------------
+
+New features:
+
+- Add a ``FilesystemMount`` model and a ``filesystem_mount`` parameter to the
+  ``LifecycleManager``. A future release will use filesystem mounts to distribute
+  content between partitions when migrating from the overlay step.
```

Comment:
```suggestion

- Add a :class:`~craft_parts.FilesystemMount` model and a ``filesystem_mount`` 
  parameter to the :class:`~craft_parts.LifecycleManager`. A future release will use 
  filesystem mounts to distribute content between partitions when migrating from the 
  overlay step.
```

Reply:
Suggested edited with @jahn-junior 's comment below

Reply:
Oh the auto-generated docs. Thanks!